.github/workflows/psalm.yml

```diff
@@ -24,7 +24,9 @@ jobs:
         uses: shivammathur/setup-php@v2
         with:
           coverage: none
-          php-version: 8.3
+          php-version: 8.4
+          ini-values: assert.exception=1, zend.assertions=1, error_reporting=-1, log_errors_max_len=0, display_errors=On
+          tools: psalm:6.7.1
```

Comment:
Temporarily locking Psalm to `6.7.1` pending resolution of https://github.com/psalm/phar/issues/19